docs/varnish/vcl/varnish5.vcl

```diff
@@ -0,0 +1,269 @@
+// Varnish VCL for:
+// - Varnish 5.1 or higher (6.0LTS recommended, and is what we mainly test against)
+//   - Varnish xkey vmod (via varnish-modules package, or via Varnish Plus)
+// - eZ Platform 2.5LTS or higher with ezplatform-http-cache (this) bundle
+//
+// Make sure to at least adjust default parameters.yml, defaults there reflect our testing needs with docker.
+
+vcl 5.1;
```

Comment:
This value is not valid, according to docs it can be either `4.0` or `4.1`: https://varnish-cache.org/docs/6.1/whats-new/upgrading-6.0.html#vcl-4-0-and-4-1

Varnish fails to starts with this VCL.

Reply:
Fixed in a335b3d31e08b4b5a3ef051fd5075d0cfa97f0df

I set it to 4.1 which seems more future proof, if you see any issues with that shout out :)